code/modules/projectiles/guns/energy/laser_vr.dm

```diff
@@ -215,19 +215,22 @@
 
 /obj/item/weapon/gun/energy/locked/frontier/carbine
 	name = "frontier carbine"
-	desc = "An ergonomically improved version of the venerable frontier phaser, the carbine is a fairly new weapon, and has only been produced in limited numbers so far. Includes a built-in crank charger for recharging away from civilization. This one has a safety interlock that prevents firing while in proximity to the facility."
+	desc = "A larger and more efficient version of the venerable frontier phaser, the carbine is a fairly new weapon, and has only been produced in limited numbers so far.  Includes a built-in crank charger for recharging away from civilization. This one has a safety interlock that prevents firing while in proximity to the facility."
 	description_fluff = "The NT Brand Model AT2 Secured Phaser System, a specialty phaser that has an intergrated chip that prevents the user from opperating the weapon within the vicinity of any NanoTrasen opperated outposts/stations/bases. However, this chip can be disabled so the weapon CAN BE used in the vicinity of any NanoTrasen opperated outposts/stations/bases. The weapon doesn't use traditional weapon power cells and instead works via a pump action that recharges the internal cells. It is a staple amongst exploration personell who usually don't have the license to opperate a lethal weapon through NT and provides them with a weapon that can be recharged away from civilization."
 	icon = 'icons/obj/gun_vr.dmi'
 	icon_state = "carbinekill"
 	item_state = "energykill"
 	item_icons = list(slot_l_hand_str = 'icons/mob/items/lefthand_guns.dmi', slot_r_hand_str = 'icons/mob/items/righthand_guns.dmi')
-	phase_power = 150
+	phase_power = 25
+	one_handed_penalty = 15 //CHOMPEdit Added this, same as phase carbine.
+	w_class = ITEMSIZE_LARGE  //CHOMPEdit Should be bigger.
```

Comment:
Personally, this makes carbines kinda eh, bc this means it can't be stored in-belt, iirc.
Usually you can do rifle on back, carbine in belt, and holdout in holster, if you wanna carry multiple. (If I'm wrong, disregard this.)